starknet_py/tests/e2e/conftest.py

```diff
@@ -314,13 +317,21 @@ def new_gateway_account_client(
     )
 
 
-@pytest.fixture(
-    scope="module",
-    params=[
+def net_to_accounts() -> List[str]:
+    accounts = [
         "gateway_account_client",
         "new_gateway_account_client",
         "rpc_account_client",
-    ],
+    ]
+    if any(net in sys.argv for net in ["--net=integration", "--net=testnet"]):
```

Comment:
Were we also supporting writing these arguments as `--net integration` or only with the `=` sign? I'd verify that.